test/molecule-role/molecule/compose/molecule.yml

```diff
@@ -9,7 +9,7 @@ lint:
     config-file: .yamllint
 platforms:
   - name: trace-java-demo
-    image: ami-0b387aea6983a6882    # Our Packer image based on Ubuntu 18.04 (EBS-Backed x86_64)
+    image: ami-09ae46ee3ab46c423    # Our Packer image based on Ubuntu 18.04 (EBS-Backed x86_64)
```

Comment:
did we regenerate our base image ?

Reply:
Yup regenerated it for the terraform version bump